addons/stock/wizard/stock_backorder_confirmation.py

```diff
@@ -37,6 +37,16 @@ def default_get(self, fields):
             # because of webclient limitations
         return res
 
+    def _show_log_less_quantities_than_expected(self, pickings):
```

Comment:
Sorry just noticed that the name of this function could be better. `show` implies that a notification (or something similar is being shown) rather than a check that does the logging as part of it's logic

```suggestion
    def _check_less_quantities_than_expected(self, pickings):
```